### ISSUE TITLE
Estimated deployment end date populated for LifeTags

### ISSUE THREAD

**Issue by jsayers-bsc** [MotusDev/Motus-TO-DO#1057]
In the Beta10 manage tags page, LifeTag deployments are given an estimated end date based on tag life. These should just be blank for LifeTags I believe?

![image](https://user-images.githubusercontent.com/55001144/144434631-f1a8e844-92ea-4596-a68b-9f402ed2762d.png)

**Comment by leberrigan** [comment]
This is still an issue

**Comment by cjardine-bsc** [comment]
Right. I was (completely arbitrarily) setting the lifespan of lifetags to 1825 days, Assuming that's essentially the max time the bird they are on will be viable (again, a very shakey assumption). 

The problem here is the provision was based on the tag Model being "LIFETAG1" but we recently changed that to just be "LifeTag" which was causing these to use the default value for lotek tags that are missing a lifespan, 400 days. 

(anyway, you don't have to care about that. Just me getting my bearings). Safe to say it's not hard for me to change things. 

The question is what do we want for CTT tags. Do we want their end dates to be NULL or do we want them to be X (5) years from deployment date?

**Comment by cjardine-bsc** [comment]
In the meantime, I put in a fix to get all the lifetags back to a lifespan of 5 years by default. We can discuss if we want Null instead.

**Comment by cjardine-bsc** [comment]
https://sandbox.motus.org/data/project/tags?projectID=0

**Comment by cjardine-bsc** [comment]
SELECT *
  FROM [motus_sandbox].[dbo].[lk_tag_models]
  where battery_model = 'SOLAR'

**Comment by bcreightonyoung** [comment]
For this issue, which displayed valued are we looking to change for the solar tags? Do we want the estimated deployment range end date, estimated lifespan and estimated buffered lifespan to all be changed, or only some of these?
